linera-service/tests/end_to_end_tests.rs

```diff
@@ -1536,11 +1565,9 @@ async fn test_end_to_end_crowd_funding() {
     let chain2 = client1.open_and_assign(&client2).await;
 
     // The players
-    let owner1 = client1.get_owner().unwrap();
-    let account_owner1 = AccountOwner::User(owner1);
-    let owner2 = client2.get_owner().unwrap();
-    let account_owner2 = AccountOwner::User(owner2);
-    // The initial accounts on chain1
+    let account_owner1 = client1.get_account_owner(); // operator
+    let account_owner2 = client2.get_account_owner(); // contributor
+                                                      // The initial accounts on chain1
```

Comment:
```suggestion

    // The initial accounts on chain1
```

(Otherwise rustfmt thinks it belongs to the previous comment.)